codelists/tests/test_presenters.py

```diff
@@ -38,3 +39,91 @@ def test_build_html_tree_highlighting_codes():
         Path(settings.BASE_DIR, "codelists", "tests", "expected_html_tree.html")
     ) as f:
         assert html.strip() == f.read().strip()
+
+
+def test_tree_tables():
+    fixtures_path = Path(settings.BASE_DIR, "coding_systems", "snomedct", "fixtures")
+    call_command("loaddata", fixtures_path / "core-model-components.json")
+    call_command("loaddata", fixtures_path / "tennis-elbow.json")
+
+    with open(fixtures_path / "disorder-of-elbow.csv") as f:
+        cl = create_codelist(csv_data=f.read())
```

Comment:
We've done these steps in a few tests now, and while I don't thinking DRYing out tests is a good thing, it might be time to think about turning these steps into fixtures?  (not in this PR though!)

Reply:
Agree.